### PR TITLE
design tweaks

### PR DESCRIPTION
**Pull request by billyroh**
updates:
* typography for mobile
* styling for buttons
* higher resolution hero image
* wider grid
* smaller waffle dudes

### CODE REVIEW

**Comment by billyroh** [comment]
@visnup: i think this works for making the grid wider? let me know if it doesn't

File: form.css [hunk 1]
```diff
@@ -29,4 +23,10 @@
     text-decoration: none;
     box-shadow: 0 2px 5px var(--light-gray);
   }
+
+  &.sm-full-width {
+    @media (--sm-viewport-down) {
+      width: 100%;
+    }
+  }
```
Comment (visnup) [inline form.css]: I'd invert this to defaulting it to `100%` and then setting it to `auto` on larger resolutions. the down-based selectors seem relatively discouraged (and can be buggy).

File: models/calendar.yml [hunk 2]
```diff
@@ -21,6 +21,7 @@
     - 8:45 PM:
         type: Talk
         emoji: 🛠
+        title: 5 Things About Chrome DevTools That Will Surprise You
```
Comment (visnup) [inline models/calendar.yml]: did he get back to us with this title already?

File: routes/index/index.css [hunk 3]
```diff
@@ -6,19 +6,22 @@
...
 
-  #hero {
+  #hero > .container {
     background: url('/images/hero-1.jpg') 50% 50%;
     background-size: cover;
-    height: 40vh;
+    height: 75vh;
+    @media (--md-viewport-down) {
+      height: 50vh;
+    }
```
Comment (visnup) [inline routes/index/index.css]: invert. mobile-first!

File: routes/index/index.jade [hunk 4]
```diff
@@ -18,7 +18,7 @@ div( class=className )
         a( href='https://goo.gl/maps/0gkOe' ) SOMA StrEat Food
         |  (in the <a href="/map" target="_self">barn</a>)
   
-  section#hero
+  section#hero: .container
```
Comment (visnup) [inline routes/index/index.jade]: aw I liked closer to full-bleed photos

File: models/calendar.yml (hunk 2 above)
Comment (billyroh) [inline models/calendar.yml]: no it's just a dumb placeholder for now. still waiting on the title and description

File: routes/index/index.jade (hunk 4 above)
Comment (billyroh) [inline routes/index/index.jade]: yeah, it's just that photos stretched beyond their intended resolution look really bad

File: form.css (hunk 1 above)
Comment (billyroh) [inline form.css]: ok done. i think it's ready for another review now

**Comment by visnup** [comment]
hero image is 500K (from 173K), which is kinda too big.

File: form.css [hunk 5]
```diff
@@ -29,4 +23,11 @@
     text-decoration: none;
     box-shadow: 0 2px 5px var(--light-gray);
   }
+
+  &.sm-full-width {
```
Comment (visnup) [inline form.css]: can you call this `.xs-full-width`? (it's only for `xs`)

File: routes/index/index.jade [hunk 6]
```diff
@@ -69,9 +69,8 @@ div( class=className )
                 span( ng-show='item.type' ) {{item.type}}
           .col-xs-12.col-lg-9( ng-show='item.person || item.type' )
             h5.highlight( ng-show='item.person' )
```
Comment (visnup) [inline routes/index/index.jade]: this doesn't have to be `.highlight` now if the name is inside the link

File: grid.css [hunk 7]
```diff
@@ -6,8 +6,10 @@
   padding-right: var(--gutter-width, 1rem);
 
   @media (--lg-viewport) {
-    padding-left: calc(6 * var(--gutter-width, 1rem));
-    padding-right: calc(6 * var(--gutter-width, 1rem));
+    padding-left: calc(10 * var(--gutter-width, 1rem));
+    padding-right: calc(10 * var(--gutter-width, 1rem));
+    max-width: 90rem;
+    width: 100%;
```
Comment (visnup) [inline grid.css]: `max-width` is cool; I'm still iffy about the `width`, but just merge now and I'll pull later and play with it in a browser.

**Comment by billyroh** [comment]
ok, i downgraded the hero image to 280kb. is that ok?

**Pull request merged**